tests/abstract_test_utils.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+from abc import ABCMeta, abstractmethod
+
+from settings import Settings
+from util.channel_access import ChannelAccessUtils
+
+
+class AbstractSingleTests(unittest.TestCase):
+    """
+    Abstract class for tests to be run exactly one regardless of how many components/configurations exist. It is meant
+    to be extended by classes for configurations and for components.
+    """
+    __metaclass__ = ABCMeta
+
+    def setUp(self):
+        """
+        This only initializes a utils object that should be instantiated as ComponentUtils or ConfigurationUtils or
+        anything else in a subclass.
+        :return: nothing.
+        """
+        self.utils = None
+
+    def test_GIVEN_an_instrument_THEN_all_block_pvs_are_interesting(self):
+        interesting_pvs = ChannelAccessUtils(Settings.pv_prefix).get_interesting_pvs()
+
+        if len(interesting_pvs) == 0:
+            print("Set of interesting PVs is empty, this is probably because the instrument {} is off. Since we do not "
+                  "know interesting pvs, {}s are not checked for non interesting block pvs test is terminated "
+                  "early.".format(Settings.pv_prefix, self.get_config_type()))
```

Comment:
I'm not sure we actually need this message at all. Other tests (e.g. `version_tests`) just silently skip the test if the instrument is off. The idea is to not "clutter up" the test output with messages that aren't errors/warnings.

You can use `self.skipTest(reason)` to indicate that the test has been skipped as opposed to "passing".